content/first-order-logic/models-theories/theories.tex

```diff
@@ -126,8 +126,8 @@
         \Part{y}{z}) \lif \Part{x}{z})]]],\\
 \intertext{Moreover, any two objects have a fusion (an object that has
   only these two objects and all their parts as parts).}  &
-\lforall[x][\lforall[y][\lexists[z][\lforall[u][(\Part{u}{z} \liff
-        (\Part{u}{x} \land \Part{u}{y}))]]]]
+\lforall[x][\lforall[y][\lexists[z][\lforall[u][(\Part{z}{u} \liff
```

Comment:
The 'and nothing else' part is not necessarily true or might be misunderstood. I would phrase it something like this (non-native speaker disclaimer): "Moreover, any two objects $x$ and $y$, for which there is an object containing both, have a sum (or fusion), that is the smallest (w.r.t 'part of') object that has $x$ and $y$ as parts."